requirements/moto.txt

```diff
@@ -2,7 +2,7 @@
 Flask==0.12
 Werkzeug==0.11.15
 click==6.7
-httpretty==0.8.10
+httpretty==0.8.14
```

Comment:
@decached some tests are failing.
Did you run the tests after upgrade?
You have to change some code accordingly.